ush/parsing_namelists_FV3.sh

```diff
@@ -0,0 +1,619 @@
+  do_mynnsfclay = ${do_mynnsfclay:-".false."}
+  icloud_bl    = ${icloud_bl:-"1"}
+  bl_mynn_edmf = ${bl_mynn_edmf:-"1"}
+  bl_mynn_tkeadvect = ${bl_mynn_tkeadvect:-".true."}
+  bl_mynn_edmf_mom = ${bl_mynn_edmf_mom:-"1"}
+  min_lakeice  = ${min_lakeice:-"0.15"}
+  min_seaice   = ${min_seaice:-"0.15"}
+EOF
+  ;;
+  FV3_GFS_v16*)
+  cat >> input.nml << EOF
+  iovr         = ${iovr:-"3"}
+  ltaerosol    = ${ltaerosol:-".false."}
+  lradar       = ${lradar:-".false."}
+  ttendlim     = ${ttendlim:-"0.005"}
+  oz_phys      = ${oz_phys:-".false."}
+  oz_phys_2015 = ${oz_phys_2015:-".true."}
+  lsoil_lsm    = ${lsoil_lsm:-"4"}
+  do_mynnedmf  = ${do_mynnedmf:-".false."}
+  do_mynnsfclay = ${do_mynnsfclay:-".false."}
+  icloud_bl    = ${icloud_bl:-"1"}
+  bl_mynn_edmf = ${bl_mynn_edmf:-"1"}
+  bl_mynn_tkeadvect = ${bl_mynn_tkeadvect:-".true."}
+  bl_mynn_edmf_mom = ${bl_mynn_edmf_mom:-"1"}
+  min_lakeice  = ${min_lakeice:-"0.15"}
+  min_seaice   = ${min_seaice:-"0.15"}
+EOF
+  ;;
+  *)
+  cat >> input.nml << EOF
+  iovr         = ${iovr:-"3"}
+EOF
+  ;;
+esac
+
+cat >> input.nml <<EOF
+  pdfcld       = ${pdfcld:-".false."}
+  fhswr        = ${FHSWR:-"3600."}
+  fhlwr        = ${FHLWR:-"3600."}
+  ialb         = ${IALB:-"1"}           ! In config.fcst
+  iems         = ${IEMS:-"1"}           ! In config.fcst
+  iaer         = $IAER			! In config.fcst
+  icliq_sw     = ${icliq_sw:-"2"}	! In config.fcst
+  ico2         = $ICO2			! In config.fcst
+  isubc_sw     = ${isubc_sw:-"2"}	! In config.fcst
+  isubc_lw     = ${isubc_lw:-"2"}	! In config.fcst
+  isol         = ${ISOL:-"2"}		! In config.fcst
+  lwhtr        = ${lwhtr:-".true."}
+  swhtr        = ${swhtr:-".true."}
+  cnvgwd       = ${cnvgwd:-".true."}
+  shal_cnv     = ${shal_cnv:-".true."}
+  cal_pre      = ${cal_pre:-".true."}	
+  redrag       = ${redrag:-".true."}
+  dspheat      = ${dspheat:-".true."}
+  hybedmf      = ${hybedmf:-".false."}
+  satmedmf     = ${satmedmf-".true."}
+  isatmedmf    = ${isatmedmf-"1"}
+  lheatstrg    = ${lheatstrg-".false."}
+  random_clds  = ${random_clds:-".true."} 
+  trans_trac   = ${trans_trac:-".true."}
+  cnvcld       = ${cnvcld:-".true."}
+  imfshalcnv   = ${imfshalcnv:-"2"}
+  imfdeepcnv   = ${imfdeepcnv:-"2"}
+  cdmbgwd      = ${cdmbgwd:-"3.5,0.25"}   
+  prslrd0      = ${prslrd0:-"0."}
+  ivegsrc      = ${ivegsrc:-"1"}
+  isot         = ${isot:-"1"}
+  lsoil        = ${lsoil:-"4"}
+  lsm          = ${lsm:-"2"}
+  iopt_dveg    = ${iopt_dveg:-"1"}
+  iopt_crs     = ${iopt_crs:-"1"}
+  iopt_btr     = ${iopt_btr:-"1"}
+  iopt_run     = ${iopt_run:-"1"}
+  iopt_sfc     = ${iopt_sfc:-"1"}
+  iopt_frz     = ${iopt_frz:-"1"}
+  iopt_inf     = ${iopt_inf:-"1"}
+  iopt_rad     = ${iopt_rad:-"1"}
+  iopt_alb     = ${iopt_alb:-"2"}
+  iopt_snf     = ${iopt_snf:-"4"}
+  iopt_tbot    = ${iopt_tbot:-"2"}
+  iopt_stc     = ${iopt_stc:-"1"}
+  debug        = ${gfs_phys_debug:-".false."}
+  nstf_name    = $nstf_name
+  nst_anl      = $nst_anl            ! In Workflow
```

Comment:
Is the "! In Workflow" comment CROW related?

Reply:
OK then, I will remove them altogether. These marks only exist in a small fraction of variables, AND could be out-of-dated, so won't be helpful anyway.